server/lib/utils.js

```diff
@@ -5,3 +5,24 @@ allowIsBoardAdmin = function(userId, board) {
 allowIsBoardMember = function(userId, board) {
   return board && board.hasMember(userId);
 };
+
+// todo XXX not really server-specific,
+// so move it to a common (client+server) lib?
```

Comment:
IIRC, it’s just historical chronology, we defined these methods before using collection-helpers.